httpx/_transports/wsgi.py

```diff
@@ -131,3 +132,20 @@ def start_response(
         stream = IteratorStream(chunk for chunk in result)
 
         return (b"HTTP/1.1", status_code, b"", headers, stream)
+
+
+class WSGIDispatch(WSGITransport):
+    def __init__(
+        self,
+        app: typing.Callable,
+        raise_app_exceptions: bool = True,
+        script_name: str = "",
+        remote_addr: str = "127.0.0.1",
+    ) -> None:
+        warnings.warn("WSGIDispatch is deprecated, please use WSGITransport")
```

Comment:
`DeprecationWarning` is missing here as well - let's review the `warnings.warn` additions to make sure we use it everywhere?

Reply:
I'll change it to use the new util function 👍